public/app/plugins/panel/graph/legend.ts

```diff
@@ -151,7 +151,11 @@ module.directive('graphLegend', function(popoverSrv, $timeout) {
 
         if (panel.legend.sort) {
           seriesList = _.sortBy(seriesList, function(series) {
-            return series.stats[panel.legend.sort];
+            let sort = series.stats[panel.legend.sort];
+            if (sort === null) {
+              sort = 0;
```

Comment:
I think we this should be -Infinity or some other big negative value so the null value is below negative values